index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+<body>
+    <header class="header">
+            <a href="#" class="logo">My Logo</a>
+            <nav>
+                <i class="material-symbols-outlined menu_icon">
+                    menu
+                </i>
+            </nav>
+    </header>
+    <div class="container">
+        <h2>I’m Abdurahim Glad to see you!</h2>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="social_media">LET’S CONNECT</p>
+        <ul class="social_media_icons">
+            <li><a href="#"><img src="images/icons/Twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/icons/Linkedin icon.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/icons/Media.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/icons/Github.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/icons/Sideshow.svg" alt="twitter"></a></li>
+        </ul>
+    </div>
```

Comment:
- [ ] I would suggest you please adjust the spacing to make your design more attractive. Please decrease the spacing of the header and also between the header and content as there is more spacing. See the image below for reference.
### Your design
![his](https://user-images.githubusercontent.com/81727988/219416335-202da2d0-ea34-4430-a2e7-199d4af53748.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/219416653-55b968fb-7f9d-48b3-a563-0c3f8d1264ac.png)